semgrep/semgrep
# adding support for running all js rules on ts files automatically.

Should fix issue https://github.com/returntocorp/enterprise/issues/427. 

Test plan: 
1. create a file helloworld.ts with code:
`let helloWorld = "test"`
2. create a file helloworld.yaml with pattern:
``` 
rules:
  - id: find-unverified-transactions
  pattern: |
    helloWorld
  message: |
    In $METHOD, there's a call to make_transaction() without first calling verify_transaction() on the Transaction object.
  languages:
  - js
  severity: WARNING
```

3. run: 
`python -m semgrep -f <path to helloworld.yaml> <directory containing helloworld.ts>`

NOTE:
* Performance is taking a hit because we are running semgrep on TS files. We should edit baseline performance in order to reflect this.
